Aquality.Selenium.Template/Aquality.Selenium.Template.NUnit/Tests/BaseTest.cs

```diff
@@ -20,6 +21,8 @@ protected string ScenarioName
         public void Setup()
         {
             Logger.Info($"Start scenario [{ScenarioName}]");
+            var customStartUp = new CustomStartUp();
```

Comment:
discussed, custom startup should be moved into the OneTimeSetUp method

Reply:
The customStartUp was moved to the OneTimeSetUp method